### DISCUSSION TITLE
Upgraded from from 6.0.8 to 6.8. Modal Window Not Displaying Correctly.

### DISCUSSION THREAD

**Discussion by jldayhoff** [RocketCommunicationsInc/astro#418] [Q&A] [answered]
Modal window does not display correctly after upgrading from 6.0.8 to 6.8. The buttons also do not have a border regardless of set to true or false. 

![image](https://user-images.githubusercontent.com/22620849/170787782-ae205dfb-f841-4366-a503-0541c2226d6b.png)

**Comment by markacianfrani** [comment, marked as answer]
@jldayhoff are you able to reproduce an example using one of our sandboxes https://astro-components.netlify.app/?path=/story/astro-uxds-welcome-start-here--page or provide a code sample showing the markup you're using?

**Reply by jldayhoff** [reply]
Hello, I am unable to reproduce it in the mock up. But here is the code: 

return (
            <div className="runs-container">
                <p>RUNS</p>
                
                {Object.entries(this.props.runs).map(([key, value])  => (
                
                <div style={{padding:2}}> 
                    
                <div style={{padding:10, display: "flex",justifyContent: "space-between", margin: "auto" , border: "2px inset blue", width: 300}}> 
                
                <rux-icon icon="playlist-add" size="small" 
                label="Add to Configuration" style={{cursor: "pointer"}} onClick={() => this.props.selectConfigRun(key)}> </rux-icon> 
                
                
                <RuxButton title={"Send to Run Description"} onClick={() => this.props.selectUtilityRun(key)} 
                borderless="true" style={{}}> {value.name} </RuxButton> 

              
                <rux-icon icon="delete-forever" size="small" label="Delete Run" style={{cursor: "pointer"}} 
                onClick={(e) => {this.showModal(key,value.name)}}> </rux-icon> 
           
                </div>

                </div>
             
                ))}
                
                <RuxModal

                open= {this.state.show}
                modalMessage= {`Are you sure you want to delete ${this.state.current_name}?`} 
                modalTitle= {`Delete ${this.state.current_name}?`}   
                style={{modalMessage: "test"}}          
              
                onRuxmodalclosed = {(e) => {this.handleModal(e)}}
                >
                 
                </RuxModal> 
       

When clicked, it produces: 

![image](https://user-images.githubusercontent.com/22620849/171212246-6c7686f4-a819-446e-9f3b-b53a68d19798.png)



It correctly rendered before I upgraded.

**Reply by jldayhoff** [reply]
My apologies. There was an older version of astro-web-components.css that was referenced. Pointed to the newest .css and I am good now. Thank You. This can be closed.

**Reply by micahjones13** [reply]
No worries, please let us know if there is anything we can do for you in the future!
